features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql/mysql5.7.sql

```diff
@@ -935,7 +935,6 @@ CREATE INDEX IDX_OROR_TID ON IDN_OIDC_REQ_OBJECT_REFERENCE(TOKEN_ID);
 CREATE INDEX IDX_ATS_TID ON IDN_OAUTH2_ACCESS_TOKEN_SCOPE(TOKEN_ID);
 
 -- SP_TEMPLATE --
-CREATE INDEX IDX_SP_TEMPLATE ON SP_TEMPLATE (TENANT_ID, NAME);
```

Comment:
any reason why this index is removed only from mysql5.7 script but not from others?

Reply:
It is removed because this query was duplicated in mysql5.7.